internal/pkg/service/buffer/storage/level/local/reader/volume/config.go

```diff
@@ -1,4 +1,4 @@
-package reader
+package volume
```

Comment:
V nasledujucich commits je to iste co bolo riesene doteraz pre `writer`, aj pre `reader`.

V tomto commite je presunutie kodu do `volume` pkg.

![image](https://github.com/keboola/keboola-as-code/assets/19371734/5ee1e87f-cd19-49ee-b8d6-3e7bbe9c13e4)